<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RebinDialog</class>
 <widget class="QDialog" name="RebinDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>327</width>
    <height>271</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Rebin the workspace</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,2">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Algorithm</string>
       </property>
       <widget class="QWidget" name="horizontalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>9</y>
          <width>291</width>
          <height>51</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,2">
         <item>
          <widget class="QLabel" name="lblAlgorithms">
           <property name="text">
            <string>Choose an algorithm:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBoxAlgorithms"/>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Bins</string>
       </property>
       <widget class="QWidget" name="gridLayoutWidget">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>9</y>
          <width>291</width>
          <height>131</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="layoutBins" columnstretch="0"/>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>RebinDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>163</x>
     <y>250</y>
    </hint>
    <hint type="destinationlabel">
     <x>163</x>
     <y>135</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>RebinDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>163</x>
     <y>250</y>
    </hint>
    <hint type="destinationlabel">
     <x>163</x>
     <y>135</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
